cloudbuild failed with tabris 3.8.0

### Problem description

I added the plugin as it is described and other plugins worked:
`  <plugin name="tabris-plugin-firebase" spec="https://github.com/eclipsesource/tabris-plugin-firebase.git" />`
This results into:
`error TS2304: Cannot find name 'firebase'.`

After that if tried to include it into package.json:
```
  "dependencies": {
    "tabris": "~3.8.0",
    "tabris-decorators": "~3.8.0",
    "tabris-plugin-firebase": "git+https://github.com/eclipsesource/tabris-plugin-firebase.git",
    "tsconfig-paths": "^3.11.0"
  },
```
This results into:
```
npm WARN tabris-plugin-firebase@4.0.0 requires a peer of tabris@3.0.x but none is installed. You must install peer dependencies yourself.
```

I cant remember what I have done, but I have also an fail with the error:
```
npm WARN tabris-plugin-firebase@4.0.0 requires a peer of tabris@3.0.x but none is installed. You must install peer dependencies yourself.
```
I think i tried to add it by npm into package-lock.json


### Expected behavior

global firebase is working with tabris 3.8.0

### Environment

- Tabris.js version: 3.8.0
- Build: cloud build

Thank you. That helped me a lot and it's running so far. But another problem is now in gradle:

```
Welcome to Gradle 6.5!
Here are the highlights of this release:
 - Experimental file-system watching
 - Improved version ordering
 - New samples
For more details see https://docs.gradle.org/6.5/release-notes.html
To honour the JVM settings for this build a new JVM will be forked. Please consider using the daemon: https://docs.gradle.org/6.5/userguide/gradle_daemon.html.
Daemon will be stopped at the end of the build stopping after processing
> Configure project :app
Warning: The 'kotlin-android-extensions' Gradle plugin is deprecated. Please use this migration guide (https://goo.gle/kotlin-android-extensions-deprecation) to start working with View Binding (https://developer.android.com/topic/libraries/view-binding) and the 'kotlin-parcelize' plugin.
FAILURE: Build failed with an exception.
* Where:
Script '/home/travis/build/eclipsesource/tabris-js-build-template/build/build/cordova/platforms/android/tabris-plugin-firebase/[secure]-build-extras.gradle' line: 2
* What went wrong:
A problem occurred evaluating project ':app'.
> Plugin with id 'com.google.gms.google-services' not found.
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
* Get more help at https://help.gradle.org
BUILD FAILED in 30s
```

Any Idea what I could do?

--- a/src/android/build-extras.gradle
+++ b/src/android/build-extras.gradle
@@ -1,3 +1,19 @@
+import com.google.gms.googleservices.GoogleServicesPlugin
+
+buildscript {
+    repositories {
+        google()
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.3.2'
+        classpath 'com.google.gms:google-services:4.3.3'
+    }
+}
+
 ext.postBuildExtras = {
-    apply plugin: 'com.google.gms.google-services'
+    if (project.extensions.findByName('googleServices') == null) {
+        apply plugin: GoogleServicesPlugin
+    }
 }

